cmd/server/handlers/handlers.go

```diff
@@ -0,0 +1,107 @@
+package handlers
+
+import (
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi/v5"
+)
+
+type Handler struct {}
+
+func NewHandler() *Handler {
+	return &Handler{}
+}
+
+func (h Handler) HomeHandler(storage *storage.MemStorage) http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		body := "<h4>Gauges</h4>"
```

Comment:
[CHANGE REQUEST] `strings.Builder`
https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go